src/modules/health/health.css

```diff
@@ -1,40 +1,34 @@
-* {
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
-}
-
 body {
-    background-image: url('content/health_background.jpg');
+    background-image: url('../../content/img/health_background.jpg');
```

Comment:
I would suggest to change this from class level. changing global (body) from css level is not good and can provide many issues in the future